src/main/java/io/kestra/plugin/azure/eventhubs/Produce.java

```diff
@@ -0,0 +1,262 @@
+                tasks:
+                  - id: readCsvFile
+                    type: io.kestra.plugin.serdes.csv.CsvReader
+                    from: "{{ inputs.file }}"
+                  - id: transformRowToJson
+                    type: io.kestra.plugin.scripts.nashorn.FileTransform
+                    from: "{{ outputs.readCsvFile.uri }}"
+                    script: |
+                      var result = {
+                        "body": {
+                          "username": row.username,
+                          "tweet": row.tweet
+                        }
+                      };
+                      row = result
+                  - id: sendToEventHubs
+                    type: io.kestra.plugin.azure.eventhubs.Produce
+                    from: "{{ outputs.transformRowToJson.uri }}"
+                    eventHubName: from-kestra
+                    connectionString: "{{ secrets.azure.eventhubs.connectionString }}"
+                    maxBatchSizeInBytes: 4096
+                    maxEventsPerBatch: 100
+                    bodySerializer: "JSON"
+                    bodyContentType: application/json
+                    eventProperties:
+                        source: kestra
+                """
+        }
+    )
+})
+@Schema(
+    title = "Publish events to Azure Event Hubs."
+)
+@Slf4j
+@SuperBuilder
+@Getter
+@NoArgsConstructor
+public class Produce extends AbstractEventHubTask implements RunnableTask<Produce.Output> {
+
+    // TASK'S METRICS
+    private static final String METRIC_SENT_EVENTS_NAME = "total-sent-events";
+    private static final String METRIC_SENT_BATCHES_NAME = "total-sent-batches";
+
+    // TASK'S PARAMETERS
+    @Schema(
+        title = "The event properties",
+        description = "The event properties which may be used for passing metadata associated with the event" +
+            " body during Event Hubs operations."
+    )
+    @Builder.Default
+    private Map<String, String> eventProperties = Collections.emptyMap();
+
+    @Schema(
+        title = "The content of the message to be sent to EventHub",
+        description = "Can be an internal storage URI, a map (i.e. a list of key-value pairs) or a list of maps. " +
+            "The following keys are supported: `from`, `contentType`, `properties`.",
+        anyOf = {String.class, List.class, Map.class}
+    )
+    @NotNull
+    @PluginProperty(dynamic = true)
+    private Object from;
+
+    @Schema(
+        title = "The hashing key to be provided for the batches of events.",
+        description = "Events with the same `partitionKey` are hashed and sent to the same partition. The provided " +
+            "`partitionKey` will be used for all the events sent by the `Produce` task."
+    )
+    private String partitionKey;
+
+    @Schema(
+        title = "The maximum size for batches of events, in bytes.",
+        description = "The maximum size to allow for the batches of events."
```

Comment:
Description is not mandatory, please avoir to repeat the title in the description.
Please remove all duplicated documentation.